alerts/ui/docker/Dockerfile

```
@@ -0,0 +1,132 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+FROM node:20-alpine as build
+
+# we need jq to read and manipulate package.json
+RUN apk add --no-cache jq
```

I would like to see some image update during build time.
```suggestion
RUN apk add --no-cache jq \
  && apk upgrade --no-cache --no-progress \
  && apk del --no-cache --no-progress apk-tools alpine-keys
```